musicdao/src/main/java/com/example/musicdao/wallet/WalletService.kt

```diff
@@ -1,14 +1,17 @@
 package com.example.musicdao.wallet
```

Comment:
This shouldn't be merged into master right? Or is it okay if we overwrite music DAO?